CentOS/RHEL script fails because $OsDiskName is empty

PedroPerezMSFT opened the pull request:
In Ubuntu/SLES instructions we use $OsDiskName throughout the whole script for the OS disk name.

In CentOS/RHEL instructions we define it as $destOsDiskName, but use $OsDiskName in Set-AzureRmVMOSDisk, which fails because $OsDiskName is of course empty.

Changed every reference to $destOsDiskNaname for $OsDiskName to fix it and also keep it consistent with the Ubuntu/SLES script.

Tested the changes: I have now deployed 2 VMs with the modified script.

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

ClarityQueen commented:
I have completed my edit pass. Sending Jaya the following questions:
I formatted this topic a bit. The Heading format was overwhelming for the questions, so I am suggesting using bold for the questions. It coordinates better with this FAQ: https://github.com/Azure/azure-content/blob/master/articles/machine-learning-faq.md. Unfortunately, Question 1 and Question 3 do not seem to want to take the ** formatting. Rather than start over, I am hoping that you can format it after you review my suggestions. I am new to this tool and not appreciating it much…sighhh.
Line 16: What does “/ tab’” mean? I think you are talking about a browser window and it is not necessary.
Line 28: Confusing sentence. Also, do you need the /# after “images”? Please verify my suggestions.
Line 7 and Line 36: “consuming” is an odd choice. Do you mean “purchasing from” or “using”?

PedroPerezMSFT committed:
$OsDiskName empty in CentOS/RHEL script

In Ubuntu/SLES instructions we use $OsDiskName throughout the whole script for the OS disk name.

In CentOS/RHEL instructions we define it as $destOsDiskName, but use $OsDiskName in Set-AzureRmVMOSDisk, which fails because $OsDiskName is of course empty.

Changed every reference to $destOsDiskNaname for $OsDiskName to fix it and also keep it consistent with the Ubuntu/SLES script.

Tested the changes: I have now deployed 2 VMs with the modified script.
